lib/livebook_web/live/session_live/secrets_component.ex

```diff
@@ -236,6 +239,11 @@ defmodule LivebookWeb.SessionLive.SecretsComponent do
        |> push_secret_selected(secret.name)}
     else
       {:error, %{errors: errors}} ->
+        errors =
+          for {name, messages} <- errors, message <- messages do
```

Comment:
the errors came back as e.g.: `%{"name" => ["is already taken"]}`. I couldn't figure out what exactly se them like that but perhaps normalising them to the changeset error shape up the stack would be the way to go?

Reply:
we could also normalize the `create_secret/2` from `Enterprise` struct to build a changeset with errors and return to be used by the `SecretsComponent`. WDYT?